feature-based-molecular-networking/README.md

```diff
@@ -1,58 +1,85 @@
-## File Formats for Different Tools
+## Feature Based Molecular Networking 
 
-### MZMine2
+For more informations on FBMN, see the workflow documentation on GNPS [https://ccms-ucsd.github.io/GNPSDocumentation/featurebasedmolecularnetworking/](https://ccms-ucsd.github.io/GNPSDocumentation/featurebasedmolecularnetworking/) and our preprint: Nothias, L.F. et al [Feature-based Molecular Networking in the GNPS Analysis Environment](https://www.biorxiv.org/content/10.1101/812404v1) bioRxiv 812404 (2019).
 
-Feature table should have three fixed columns
+Representative input files for each supported tools are available at:
+[https://github.com/CCMS-UCSD/GNPS_Workflows/tree/master/feature-based-molecular-networking/test/reference_input_file_for_formatter](https://github.com/CCMS-UCSD/GNPS_Workflows/tree/master/feature-based-molecular-networking/test/reference_input_file_for_formatter)
 
-1. Row ID
-2. Row m/z
-3. Row retention time
+Formatter script for each tool supported can be found in:
+[https://github.com/lfnothias/GNPS_Workflows/tree/master/feature-based-molecular-networking/tools/feature-based-molecular-networking/scripts](https://github.com/lfnothias/GNPS_Workflows/tree/master/feature-based-molecular-networking/tools/feature-based-molecular-networking/scripts)
```

Comment:
Here are the absolute links in the README

Reply:
Thank you